src/content/docs/en/pages/guides/marketplace/templates/devscard.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: How to deploy DevsCard using a template
+description: >-
+  This template contains the configurations to create your online and print resume using the DevsCard project.
+meta_tags: >-
+  templates, guides, Azion Marketplace, resume, portfolio, DevsCard, web, print
+namespace: docs_guides_devscard
+permalink: /documentation/products/guides/devscard/
+---
+
+import Tag from 'primevue/tag';
+
+<Tag severity="info" client:only="vue">
+Preview
+</Tag>
+
+The DevsCard template is an initial solution for creating and managing your online and print resume using the DevsCard project. After deployment, you can easily update the content with your personal information and develop your portfolio.
+
+The deployment of this template creates a GitHub repository containing your project, as well as an edge application and a domain to facilitate your access and management through Azion Edge Platform.
+
+This template uses version 2.8.0 of Astro.
+
+---
+
+## Requirements
+
+- A [GitHub account](https://github.com/signup) to connect with Azion and create your new repository.
+    - Every push will be deployed automatically to this repository to keep your project updated.
+- Enable [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), and [Edge Cache](/en/documentation/products/build/edge-application/edge-cache/) in your Azion account.
+    - To do so, go to the [Billing & Subscriptions](/en/documentation/products/guides/billing-and-subscriptions/#subscriptions) section and activate the switch for each module.
+    - If these modules aren't activated, the execution will fail and a log explaining the reason will be printed.
+    - If these modules are activated, deploying this template could generate usage-related costs. Check the [pricing page](/en/documentation/products/pricing/) for more information.
```

Comment:
```suggestion
- This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
```